src/workerd/api/actor-state.c++

```diff
@@ -713,6 +714,61 @@ jsg::Promise<jsg::Value> DurableObjectState::blockConcurrencyWhile(jsg::Lock& js
   return IoContext::current().blockConcurrencyWhile(js, kj::mv(callback));
 }
 
+void DurableObjectState::acceptWebSocket(
+    jsg::Ref<WebSocket> ws,
+    jsg::Optional<kj::Array<kj::String>> tags) {
+  JSG_ASSERT(!ws->isAccepted(), Error,
+      "Cannot call `acceptWebSocket()` if the WebSocket was already accepted via `accept()`");
+  JSG_ASSERT(ws->LOCAL == WebSocket::Locality::LOCAL, Error,
```

Comment:
Just realized this does absolutely nothing, and also we don't even want it to be local because of the `couple()` stuff I mentioned.